.gitlab-ci.yml

```
@@ -82,6 +83,15 @@ integration:
         --network test
         test_integration_go
 
+### Checking checked-in generated code is up to date
+
+protogen:
+  stage: gencode
+  image: nixos/nix
+  script:
+    - apk add git
```

Let's use Nix instead:

```suggestion
    - nix-env -iA nixpkgs.git
```